projects/aca-playwright-shared/src/page-objects/components/search/search-input.component.ts

```diff
@@ -29,6 +29,7 @@ import { timeouts } from '../../../utils';
 export class SearchInputComponent extends BaseComponent {
   private static rootElement = 'aca-page-layout';
   public searchButton = this.getChild('aca-search-input .app-search-button');
+  getIconByName = (name: string): Locator => this.getChild('.mat-icon[role="img"]', { hasText: name });
```

Comment:
you don't need role in the selector probably

Reply:
removed role from selector.